run_integration_tests.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+IGNORE_HEADER="// INTEGRATION TESTS IGNORE"
+TEMP_TEST_CASE_PREFIX="test_case"
+TEMP_SPLIT_TEST_CASE_PREFIX="current_case"
+
+FAILED_CASES=0
+FAILED_TO_COMPILE=0
+
+# bash trickery to get rid of identations at the start of lines
+print_multiline_message() {
+    echo "$1" | sed 's/^[ \t]*//g'
+}
+
+run_test_case() {
+    test_number=$(echo $1 | grep -Eo '[0-9]+$')
+    echo "Case number $test_number..."
```

Comment:
I think it's too verbose to print a new line for every successful test. To indicate progress we can just output the number of tests passed with `\r`.